test/integration/test_integration.py

```diff
@@ -2028,30 +2028,14 @@ def test_get_model_size_aqt(self, api, test_device, test_dtype):
 
 
 class TestBenchmarkModel(unittest.TestCase):
-    class ToyLinearModel(torch.nn.Module):
-        def __init__(self, m=64, n=32, k=64):
-            super().__init__()
-            self.linear1 = torch.nn.Linear(m, n, bias=False)
-            self.linear2 = torch.nn.Linear(n, k, bias=False)
-
-        def example_inputs(self, batch_size=1, dtype=torch.float32, device="cpu"):
-            return (
-                torch.randn(
-                    batch_size, self.linear1.in_features, dtype=dtype, device=device
-                ),
-            )
-
-        def forward(self, x):
-            x = self.linear1(x)
-            x = self.linear2(x)
-            return x
+    from torchao.testing.model_architectures import ToyTwoLinearModel
 
     def run_benchmark_model(self, device):
         # params
         dtype = torch.bfloat16
-        m = self.ToyLinearModel(1024, 1024, 1024).eval().to(dtype).to(device)
+        m = self.ToyTwoLinearModel(1024, 1024, 1024).eval().to(dtype).to(device)
```

Comment:
this seems wrong, you'll need to pass `dtype` and `device` to ToyTwoLinearModel I think, also  no need to get it from `self` I think

does the test pass locally?